scripts/makefile/tests.mk

```diff
@@ -56,7 +56,8 @@ endif
 ## Validate composer.json file
 compval:
 	@echo "Composer.json validation..."
-	@docker run --rm -v `pwd`:`pwd` -w `pwd` $(IMAGE_PHP) composer validate --strict
+	# Can't use --strict cause we need dev versions for d9 compatibility
```

Comment:
Need to remove it from there too https://github.com/skilld-labs/skilld-docker-container/blob/d9/.gitlab-ci.yml#L49

Reply:
Fixed